controllers/rbac/manager.go

```diff
@@ -115,36 +121,42 @@ func (r *Manager) Reconcile(ctx context.Context, request reconcile.Request) (res
 }
 
 func (r *Manager) EnsureClusterRoleBinding() (err error) {
-	crb := &rbacv1.ClusterRoleBinding{
-		ObjectMeta: v1.ObjectMeta{
-			Name: ProvisionerRoleName,
-		},
-	}
-
-	_, err = controllerutil.CreateOrUpdate(context.TODO(), r.Client, crb, func() error {
-		// RoleRef is immutable, so we need to delete and recreate ClusterRoleBinding if it changed
-		if crb.ResourceVersion != "" && !equality.Semantic.DeepDerivative(provisionerClusterRoleBinding.RoleRef, crb.RoleRef) {
-			return ImmutableClusterRoleBindingError{}
-		}
-		crb.RoleRef = provisionerClusterRoleBinding.RoleRef
-		crb.Subjects = []rbacv1.Subject{
-			{
-				Kind: "Group",
-				Name: r.CapsuleGroup,
+	for _, group := range r.CapsuleGroups {
+		name := b64.RawStdEncoding.EncodeToString([]byte(group))
+		crb := &rbacv1.ClusterRoleBinding{
+			ObjectMeta: v1.ObjectMeta{
+				Name: fmt.Sprintf("%s-%v", ProvisionerRoleName, name),
+				Labels: map[string]string{
+					rbacLabel: "",
```

Comment:
What about to use `capsule.clastix.io/rbac` as key and the Object name as value?

Reply:
done